lidiabrowser/sync/populate.py

```diff
@@ -0,0 +1,210 @@
+from typing import Optional
+from django.db import transaction
+
+import yaml
+import logging
+
+import sync.models as syncmodels
+from sync.zoteroutils import get_attachment_url, get_attachment_id_from_url
+from lidia.models import ArticleTerm, BaseAnnotation, Category, ContinuationAnnotation, LidiaTerm, Publication as LidiaPublication, TermGroup
+from lidia.models import Annotation as LidiaAnnotation
+from lidia.models import Language  #, ArticleTerm, LidiaTerm, Category
+
+logger = logging.getLogger(__name__)
+
+
+LIDIAPREFIX = "~~~~LIDIA~~~~"
+
+
+def process_continuation_annotations() -> None:
+    # Get all annotations in order of presence in documents
+    annotations = BaseAnnotation.objects.order_by(
+        "parent_attachment", "sort_index"
+    )
+    current_parent = None
+    current_first_annotation = None
+    to_be_deleted = []
+    for annotation in annotations:
+        if hasattr(annotation, "annotation"):
+            # Object is of Annotation model (first annotation)
+            current_first_annotation = annotation.annotation  # noqa
```

Comment:
Do we need to change something with the linter config? Or why is it needed?

Reply:
Sorry, this should have been `# type: ignore`. I am using Pyright and it does not recognize `hasattr`.